src/app/admin/generator/page.tsx

```
@@ -0,0 +1,409 @@
+            <div className="grid gap-6 lg:grid-cols-12 items-start">
+                {/* Left Column: Controls */}
+                <div className="space-y-6 lg:col-span-7 xl:col-span-8">
+                    {/* Style & Tone Selection */}
+                    <Card>
+                        <CardHeader>
+                            <CardTitle>Configuration</CardTitle>
+                            <CardDescription>Choose the personality of the question.</CardDescription>
+                        </CardHeader>
+                        <CardContent className="space-y-6">
+                            <div className="space-y-3">
+                                <label className="text-sm font-medium">Style</label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)